### DISCUSSION TITLE
Great features that need to be in Dota 2 plus or Dota 2 Labs

### DISCUSSION THREAD

**Discussion by 3nines** [ValveSoftware/Dota2-Gameplay#16347] [Ideas]
I have had these ideas for a very long time, have now had the opportunity to put them all together, hope you like it. I have compiled these ideas, with detailed descriptions and justifications for how each suggestion can contribute to improving the game.

**It is up to Valve to decide whether these features should be integrated into Dota Plus or tested in Dota 2 Labs.**

<br>
<br>

**1. Display of teammate's bottle status at runes (Hero Icon Hover on Runes for Bottle).**

This proposal introduces a new feature that allows players to directly see the status of their teammates' bottles when hovering over a rune on the map. By implementing two visual indicators - one icon that becomes visible when a bottle icon is present and another that shows the number of fillings from 3 to 0 - players can more easily determine who would benefit most from the rune. This reduces the risk of misunderstandings and optimizes the use of valuable resources, leading to a more cohesive and strategic gameplay experience.

![753shots_so](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/6ea6bba3-f5d8-4eaf-884b-c46e36dd73a9)

<br>
<br>

**2. Display the number of times a teammate has stacked.**

Now you can see how many times your Hard Support or anyone in your team has stacked in the map, with the Hero icon next to it. This sends positive vibes to your teammates and thanks them for the teamwork effort they have made.

You can also see it on the minimap where the creeps have been stacked. This makes it easier for players to see where there is a lot of farm to be collected.

![699shots_so](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/4fafff0d-659a-4b83-a6fe-c684c0fdb4ba)

<br>
<br>

**3. Preselected positions for wards.**

There should be preselected positions for placing wards across the map where you have chosen to place them beforehand, then saved over the cloud for your eyes only, your teammates can't see your ward positions. These ward positions can either be seen by holding the "ALT" key or you can choose to have them always visible.

The advantage is that sometimes you try hiding your sentry ward from the opponent. Often, you try not to block a tree or a box of a camp; you always try to stand as close to it as possible to cover as much area of the sentry you're about to place. Now, you might only need to do this once, and then simply drag your wards over the predefined position. See the image below for a demo.

![182shots_so](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/a2edc61b-e4fd-43f4-91a7-bce9c3efc899)

<br>
<br>

**4. Show two more items, on Head-up display holding ALT**

Display two extra item slots out of 12 Consumable items, exclusively containing utility items for you and your teammates. Each player can pre-select the items they want to display to their team. If they haven't chosen, the system will automatically select if someone possesses any of these 12 consumable items. See the image below for a demo.

For instance, you can click on the "Smoke" icon if someone in your team has it, instead of having to click on their profile and then navigate to their inventory.

![858shots_so](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/6b88ef9e-97cb-4772-ac85-71cb19416c49)

<br>
<br>

**5. Talent Tree 3 second cooldown**

I can't count how many times one has accidentally selected the wrong Talent Tree level. Now, you can instantly reset to the previous level within 3 seconds if the wrong choice is made. After you've made the correct selection, the Talent you've chosen will be on a cooldown for 5 seconds.

This could be further improved by locking both sides and, instead of choosing, a 3-second cooldown appears in the middle, just like in the demo image below, and then you make the correct selection by first clicking on the 3-second cooldown, then choose the right one. This could perhaps be offered as an optional setting.

![29shots_so](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/de72194c-972e-432c-a1e8-eec0394d661a)

<br>
<br>

///

I have a lot more ideas that I think will help us improve the game and gameplay. But let's see if there is time to present them.

Swim safe my friends. 

See you.

**Comment by commentator24** [comment]
1. seems a bit too much to just be able to see empty/full bottles. but might be nice if theres UI to toggle / see teammates inventory cooldowns, empty/full bottles, including their courier inventory to see if theres items coming before teamfights.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/16937876-5018-42de-9841-874b41e3fa59)

2. seems a bit too much again to just see stacks, but might be nice if minimap show bigger icon/more layers/changing colour e.g. orange become redder to show the difference when camp is stacked.

3. i think with experience you'll just remember which spots are good. there are also markers on map to show good positions, like lamps or darker grass/bush.

4. this seems very nice, especially if able to customise which skills/items to see. e.g. ultimate/items cooldowns timer cd. 

5. instead of 3 sec cooldown i would love a feature to undo talents and skills, so players can quickly plan and adapt to changing situations. too many games we check teammates skills and see they didnt level up certain skills and have to wait for them level up. Game is over by then because of 1 teammate. maybe this can also trap cheaters who want to abuse the system when they quickly swap skills.
